app/models/concerns/political.rb

```diff
@@ -0,0 +1,32 @@
+module Political
```

Comment:
Similarly here, you can put this concern in a separate commit and combine it with https://github.com/alphagov/frontend/pull/4573/commits/5996012547f39f30f6bf34674e18656574efebb7